src/main/java/com/SocialMedia/SocialMedia/models/dto/UserDTO.java

```diff
@@ -1,4 +1,22 @@
 package com.SocialMedia.SocialMedia.models.dto;
 
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDate;
+@Getter
+@Setter
+
 public class UserDTO {
+
+    public String firstName;
+    public String lastName;
+    public String email;
+    public String username;
```

Comment:
- use camelCase for naming of all fields;

Reply:
in this case there is no need for camelCase